sdk/lib/3rdparty/freetype/freetype.def

```diff
@@ -23,6 +23,7 @@ EXPORTS
   FT_Attach_File
   FT_Attach_Stream
   FT_Bitmap_Convert
+  FT_Bitmap_Convert_ReactOS_Hack
```

Comment:
If we don't use that .def file (I doubt we do because we use spec files), I would suggest you simply omit adding the _Hack entry there.

Reply:
```suggestion
```